### PR TITLE
fixing 2 tests for rgw

### PR DESCRIPTION
**Pull request by archanabpatil**
Signed-off-by: Archana Patil <archana.patil@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [x] New/Affected tests are executed on Latest Build
-  [x] Attach test execution logs
-  [x] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: tests/s3/test_multipart_getput.py [hunk 1]
```diff
@@ -316,19 +322,28 @@ def test_multipart_upload_test_28538(self):
         self.log.info("Listed parts of multipart upload: %s", res[1])
         self.log.info(" Complete the multipart with first and last part upload")
         uploaded_parts[1].reverse()
-        try:
-            resp = self.s3_mpu_test_obj.complete_multipart_upload(mpu_id, uploaded_parts[1],
+        if S3_ENGINE_RGW == CMN_CFG["s3_engine"]:
```
Comment (seagate-sarang-sawant) [inline tests/s3/test_multipart_getput.py:325]: Are the multi part upload APIs different for RGW from standard multipart low level APIs?

File: tests/s3/test_multipart_getput.py [hunk 2]
```diff
@@ -316,19 +322,28 @@ def test_multipart_upload_test_28538(self):
...
-            self.log.error(error)
-            assert_s3_err_msg(errmsg.RGW_ERR_WRONG_JSON, errmsg.CORTX_ERR_WRONG_JSON,
-                              CMN_CFG["s3_engine"], error)
-            self.log.info("Failed to complete the multipart with incomplete part details ")
-        self.log.info("Aborting multipart uploads")
-        self.s3_mpu_test_obj.abort_multipart_upload(self.bucket_name,
-                                                    self.object_name, mpu_id)
+                assert_utils.assert_true(resp[0], resp[1])
+            except CTException as error:
+                self.log.error(error)
```
Comment (seagate-sarang-sawant) [inline tests/s3/test_multipart_getput.py:332]: If you run into exception in this except,  you would break the behavior of test.

**Review by seagate-sarang-sawant — CHANGES_REQUESTED** [review]

File: tests/s3/test_multipart_getput.py (hunk 1 above)
Comment (archanabpatil) [reply tests/s3/test_multipart_getput.py:325]: No, they are same

File: tests/s3/test_multipart_getput.py (hunk 2 above)
Comment (archanabpatil) [reply tests/s3/test_multipart_getput.py:332]: I modified this , and as we discussed, will handle the RGW related filtering in next PI

**Review by apoorva-rao — APPROVED** [review]

**Review by seagate-sarang-sawant — APPROVED** [review]

**Review by ketanarlulkar — APPROVED** [review]

**Pull request merged**